.github/workflows/release.yml

```diff
@@ -15,54 +15,21 @@ name: Release
 jobs:
   build:
     uses: ./.github/workflows/continuous-integration.yml
-  upload:
```

Comment:
What happened to all of this?  Aren't the npm packages for shell binaries also being uploaded?

Reply:
This PR is targeting a feature branch only to test that the build/pack output is correctly in place. This version will not ever be merged to the main branch. I removed all the npm stuff to prevent uploading npm packages and the zip file during the test run.